blob/fileblob/fileblob.go

```diff
@@ -15,24 +15,26 @@
 // Package fileblob provides a blob implementation that uses the filesystem.
 // Use OpenBucket to construct a *blob.Bucket.
 //
+// Blob keys are escaped before being used as filenames, and filenames are
```

Comment:
Looks like you added this couple of paragraphs back; please delete (it's replaced by the `Escaping` section below).

Reply:
Done.